docs/source/features/sheet_management/schema.rst

```diff
@@ -95,9 +95,6 @@ An enum type can be created by giving the values:
    >>> enum.values
    ['a', 'b', 'c']
 
-.. note::
```

Comment:
do we need to add description about enum schema like we need to use int value in data?

Reply:
Users dont't need to use int value in data.